flycheck.el

```diff
@@ -6450,26 +6450,23 @@ the BUFFER that was checked respectively.
 
 See URL `https://palantir.github.io/tslint/' for more information
 about TSLint."
-  (let ((json-array-type 'list))
```

Comment:
Was this somehow redundant with the former `json-read`?

Reply:
Yup, exactly.  flycheck-parse-json already sets this variable